source/fundamentals/write-data.txt

```diff
@@ -37,13 +37,35 @@ an application configured to use the {+provider-short+}.
    To learn how to configure an application to use the {+provider-short+}, see
    :ref:`entity-framework-configure`.
 
+Transactional Write Operations
+-------------------------------
+
+The ``SaveChanges()`` and ``SaveChangesAsync()`` methods are transactional by
+default. This means that if an error occurs during an operation, the provider
+rolls back any changes made to the database. Because of this, your application
+must be connected to a transaction-capable deployment of MongoDB server, such as
+a replica set.
+
+You can disable automatic transactions in the ``SaveChanges()`` and
+``SaveChangesAsync()`` methods by setting the ``AutoTransactionBehavior`` property to
+``AutoTransaction.Never`` on your ``DbContext`` subclass during application
+setup. However, we do not recommend disabling this feature. Doing so causes
+any concurrency changes or operation failures during the save operation
+to leave the database in an inconsistent state.
+
+The following example shows how to disable automatic transactions in the
```

Comment:
Maybe prefix with "If, however, you choose to do this" for people speed-reading/scanning next to not just land here and think the answer is to disable transactions? Hopefully will send them back up to the previous paragraph.

Reply:
Added a warning for emphasis as this aligns more with our typical style